cluster/gce/util.sh

```diff
@@ -950,6 +1136,10 @@ $(echo "${CUSTOM_CALICO_NODE_DAEMONSET_YAML:-}" | sed -e "s/'/''/g")
 CUSTOM_TYPHA_DEPLOYMENT_YAML: |
 $(echo "${CUSTOM_TYPHA_DEPLOYMENT_YAML:-}" | sed -e "s/'/''/g")
 EOF
+  if [[ "${master}" == "false" ]]; then
```

Comment:
Why is this needed?

Reply:
I suspect there used to be something in this block but then it was removed. I've deleted it, cc @mtaufen in case there's some reason for it.